templates/js-template-default/frameworks/runtime-src/proj.android-studio/gradle.properties

```diff
@@ -17,10 +17,26 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
-PROP_TARGET_SDK_VERSION=22
+# Android SDK version that will be used as the compile project
+PROP_COMPILE_SDK_VERSION=27
+
+# Android SDK version that will be used as the earliest version of android this application can run on
+PROP_MIN_SDK_VERSION=16
+
+# Android SDK version that will be used as the latest version of android this application has been tested on
+PROP_TARGET_SDK_VERSION=27
+
+# Android Build Tools version that will be used as the compile project
+PROP_BUILD_TOOLS_VERSION=27.0.3
+
+# List of CPU Archtexture to build that application with
+# Available architextures (armeabi-v7a | arm64-v8a | x86)
+# To build for multiple architexture, use the `:` between them
+# Example - PROP_APP_ABI=armeabi-v7a:arm64-v8a:x86
 PROP_APP_ABI=armeabi-v7a
 
-RELEASE_STORE_FILE=
-RELEASE_STORE_PASSWORD=
-RELEASE_KEY_ALIAS=
-RELEASE_KEY_PASSWORD=
+# uncomment it and fill in sign information for release mode
+RELEASE_STORE_FILE=/Users/laptop/code-github/fireball/editor/static/build-templates/native/debug.keystore
```

Comment:
?? Hardcode for store file?

Reply:
my god, it's a mistake. I will re-check it tomorrow.